index.js

```diff
@@ -13,6 +13,16 @@
       random: false
     }, options);
 
+    if (Array.isArray(input)) {
```

Comment:
How about checking `Array.isArray(input)` inside of `isArrayOfStrings` function.

I thought about this solution for checking array of strings
```js
const isArrayOfStrings = (input) => {
    if(Array.isArray(input)) {
      input.forEach((value, i) => {``
        if (typeof value !== 'string') throw TypeError(`Expected array of strings but got type '${typeof value}' at  index ${i}`);
      });
      // If no error found returns true
      return true;
    }
  }
```
and call it just before `return input.replace( ...` like:
```js
if(isArrayOfStrings(input)) input = input.join('');
```
What do you think?!